### ISSUE TITLE
Add ability to keyboard tab to open main nav dropdown on truss.works.

### ISSUE THREAD

**Issue by xtine** [trussworks/accessibility#2]
Issue 6: Tabbing to open menus
Keyboard users should be able to navigate the menu. While this is technically possible (if they look at the destination URL in the bottom left of their browser window), the menus do not open visually, which makes it hard to know where they are in the navigation
Doc with all documented accessibility issues:
https://docs.google.com/document/d/1upM9ue4UAuJwIBbZ6fRdjGKmzs-odcWxDQfYftIrHyw/edit?usp=sharing

PT IMPORTED STORY: https://www.pivotaltracker.com/story/show/171138619

**Comment by xtine** [comment]
Getting tab focus to work within the constraints of our Squarespace theme proves difficult without JavaScript code injection (because we cannot easily edit the underlying HTML/CSS) which I would not recommend in this circumstance.
